### DISCUSSION TITLE
Running Error in NFT Minting Dapp template

### DISCUSSION THREAD

**Discussion by CryptainGolD** [aptos-labs/aptos-developer-discussions#447] [Dev Doc and Tutorial Feedback]
After successfully running of `npm run move:publish`

Transaction submitted: https://explorer.aptoslabs.com/txn/0x4f7c948a30a486ef0ea8b161702dd60d6885f8ff5e31c0e95b692265fd824b1e
Code was successfully deployed to object address 0x530e2d1c61b5995defc04f4b68dce32715f7021b63e8a20d745151596127b068
{
  "Result": "Success"
} 

I ran the command: `npm run dev`, so

> nft-minting-dapp-template@0.0.0 dev
> vite

The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.

  VITE v5.4.6  ready in 2090 ms

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
  ➜  press h + enter to show help

But http://localhost:5173/ can't be reached.

**Comment by 0xmaayan** [comment]
It might be related to the solution on your other thread https://github.com/aptos-labs/aptos-developer-discussions/discussions/446 and you are having versioning issues.

The output is not an error but a warning so it should not block the app to run, try deleting `package-lock.json` and `node_modules` and then run `npm install`

**Reply by CryptainGolD** [reply]
I've already updated npm version as latest version. I always delete `package-lock.json` and reinstall npm

```
"dependencies": {
    "@aptos-labs/ts-sdk": "^1.28.0",
    "@aptos-labs/wallet-adapter-react": "^3.5.0",
    "@irys/sdk": "^0.2.1",
    "@radix-ui/react-accordion": "^1.1.2",
    "@radix-ui/react-alert-dialog": "^1.0.5",
    "@radix-ui/react-aspect-ratio": "^1.0.3",
    "@radix-ui/react-collapsible": "^1.0.3",
    "@radix-ui/react-dialog": "^1.0.5",
    "@radix-ui/react-dropdown-menu": "^2.0.6",
    "@radix-ui/react-label": "^2.0.2",
    "@radix-ui/react-popover": "^1.0.7",
    "@radix-ui/react-progress": "^1.0.3",
    "@radix-ui/react-separator": "^1.0.3",
    "@radix-ui/react-slot": "^1.0.2",
    "@radix-ui/react-toast": "^1.1.5",
    "@radix-ui/react-tooltip": "^1.0.7",
    "@tanstack/react-query": "^5.40.1",
    "class-variance-authority": "^0.7.0",
    "clsx": "^2.1.1",
    "date-fns": "^3.6.0",
    "lucide-react": "^0.383.0",
    "react": "^18.2.0",
    "react-day-picker": "^8.10.1",
    "react-dom": "^18.2.0",
    "react-router-dom": "^6.23.1",
    "tailwind-merge": "^2.3.0",
    "tailwindcss-animate": "^1.0.7"
  },
  "devDependencies": {
    "@types/node": "^20.14.1",
    "@types/react": "^18.2.66",
    "@types/react-dom": "^18.2.22",
    "@typescript-eslint/eslint-plugin": "^7.2.0",
    "@typescript-eslint/parser": "^7.2.0",
    "@vitejs/plugin-react": "^4.2.1",
    "autoprefixer": "^10.4.19",
    "crypto-browserify": "^3.12.0",
    "dotenv": "16.3.1",
    "eslint": "^8.57.0",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.4.6",
    "js-yaml": "4.1.0",
    "os-browserify": "^0.3.0",
    "path-browserify": "^1.0.1",
    "postcss": "^8.4.38",
    "prettier": "^3.3.2",
    "stream-browserify": "^3.0.0",
    "tailwindcss": "^3.4.3",
    "tree-kill": "1.2.2",
    "typescript": "^5.2.2",
    "vercel": "^35.2.4",
    "vite": "^5.2.0",
    "vite-plugin-node-polyfills": "^0.21.0",
    "vite-plugin-notifier": "^0.1.5"
  }
```

**Reply by 0xmaayan** [reply]
yes, but there might be something going on with npm trying to resolve packages, this is why you could try deleting `package-lock.json` and `node_modules` and then run `npm install` on the root folder.

I wasn't able to reproduce this issue, so it might be something with your env

**Reply by CryptainGolD** [reply]
I've resolved this issue from stackoverflow.com
Rewrite` "dev": "vite --host"` in `package.json`
Or add `host: true` into server data in `vite.config.ts`

```
export default defineConfig({
  build: {
    outDir: "dist",
  },
  server: {
    open: true,
    host: true,
  },
...
```

It managed work successfully
